core/src/main/java/org/fao/geonet/kernel/search/SearchManager.java

```diff
@@ -334,23 +334,20 @@ private Analyzer createAnalyzerFromLuceneConfig(final String analyzerClassName,
                         Constructor<? extends Analyzer> c = analyzerClass.getConstructor(clTypesArray);
                         analyzer = c.newInstance(inParamsArray);
                     } catch (Exception x) {
-                        SE_LOGGER.warn("   Failed to create analyzer with parameter: {}",x.getMessage());
-                        x.printStackTrace();
+                        SE_LOGGER.warn("   Failed to create analyzer with parameter: {}",x.getMessage(), x);
```

Comment:
SE_LOGGER is a log4j logger !
Affraid all the uppercase one are also (because static).

Reply:
@cmangeat Not sure what is the issue, the refactor haven't change the code to use a log4j logger, just added the exception to the logger method instead of using the `printStackTrace`

Reply:
I was considering the slf4j api:

debug(String msg); 
debug(String format, Object arg); 
debug(String format, Object arg1, Object arg2);           
debug(String msg, Throwable t);

, for example,

I am not sure you can both use substitution mechanism debug("{}gnagna", "ra") for example, and exception log mechanism at the same time  debug("{}gnagna", "ra", ex).

I even don't what "debug(String msg, Throwable t)" does.

Reply:
This said, not sure we really want the stacktrace to be logged...